Gruntfile.js

```diff
@@ -164,7 +146,7 @@ module.exports = (grunt) => {
                 force: false
             },
             files: {
-                src: trans2TSFiles
+                src: TSFiles
```

Comment:
Re globs, because they only go one level deep, unfortunately. That is, `src/**/*.ts` picks up `src/xxx/*.ts`, but does not pick up `src/*.ts` nor `src/xxx/yyy/*.ts`. If there's a way to get it to go deep, do let me know.

But even there is a way, or if we were to do
```js
tslint.files.src = [
    'test/*.ts',
    'src/*.ts',
    'src/**/*.ts',
    'src/**/**/*.ts',
    'src/**/**/**/*.ts'
];
```
...we also need to filter the initial results (removing ends-with `.d.ts` and contains `/notifications/`) and I could not get `tslint.options.exclude` to work at all (with a string, much less with a string array).